src/api/entities/Instruction/index.ts

```diff
@@ -181,15 +195,15 @@ export class Instruction extends Entity<UniqueIdentifiers> {
 
     const status = meshInstructionStatusToInstructionStatus(rawStatus);
 
-    if (status === InstructionStatus.Unknown) {
+    if (status === InternalInstructionStatus.Unknown) {
       throw new PolymeshError({
         code: ErrorCode.FatalError,
         message: notPendingMessage,
       });
     }
 
     const details = {
-      status,
+      status: InstructionStatus.Pending,
```

Comment:
Why is the explicit assignment necessary? Isn't it enough to use `status`?

Reply:
`status` variable's type is `InternalInstructionStatus` which represents `Pending` or `Unknown`. 
`status` in details is `InstructionStatus` which represents `Pending`, `Executed` or `Failed`. 
I had to choose to force `Pending` typing or this explicit assignment which sounds like the best option for me.